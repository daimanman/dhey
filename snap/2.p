{"C":10,"Z":"4s","N":10,"Remark":"TEST TASK","Url":"","Method":"post","TaskId":2,"Type":"","FileId":"3","Status":0,"Err":"","QH":{}}