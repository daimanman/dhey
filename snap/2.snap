{"AvgTotal":0.8953878000000001,"Fastest":0.0561352,"Slowest":0.152408,"Average":0.08953878000000001,"Rps":11.168345157260353,"AvgConn":0.00491308,"AvgDNS":0.00010054000000000001,"AvgReq":0.00010016999999999998,"AvgRes":0.030832609999999993,"AvgDelay":0.05369292,"ConnMax":0,"ConnMin":0.0010054,"DnsMax":0,"DnsMin":0.0010054,"ReqMin":0.0010017,"ReqMax":0,"ResMin":0.0010017,"ResMax":0.0039604,"DelayMax":0.0521748,"DelayMin":0.0552347,"Lats":[0.152408,0.1042781,0.1042782,0.1023024,0.103245,0.1002773,0.0561352,0.0561538,0.0581849,0.0581249],"ConnLats":[0.0010054,0,0,0,0,0,0,0,0,0],"DnsLats":[0.0010054,0,0,0,0,0,0,0,0,0],"ReqLats":[0,0,0,0,0,0,0,0,0.0010017,0],"ResLats":[0.0500379,0.05006,0.0491305,0.0480909,0.0480103,0.0470205,0.0039604,0.0039611,0.004042,0.0040125],"DelayLats":[0.0532393,0.0542181,0.0551477,0.0542115,0.0552347,0.0532568,0.0521748,0.0521927,0.0531412,0.0541124],"Total":895387800,"ErrorDist":{},"StatusCodeDist":{"200":10},"SizeTotal":0,"SizeReq":0,"NumRes":10,"LatencyDistribution":[{"Percentage":10,"Latency":0.0561538},{"Percentage":25,"Latency":0.0581849},{"Percentage":50,"Latency":0.1023024},{"Percentage":75,"Latency":0.1042782},{"Percentage":90,"Latency":0.152408},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.0561352,"Count":1,"Frequentcy":0.1},{"Mark":0.06576248,"Count":3,"Frequentcy":0.3},{"Mark":0.07538976,"Count":0,"Frequentcy":0},{"Mark":0.08501704,"Count":0,"Frequentcy":0},{"Mark":0.09464432,"Count":0,"Frequentcy":0},{"Mark":0.10427159999999999,"Count":3,"Frequentcy":0.3},{"Mark":0.11389888,"Count":2,"Frequentcy":0.2},{"Mark":0.12352616,"Count":0,"Frequentcy":0},{"Mark":0.13315343999999998,"Count":0,"Frequentcy":0},{"Mark":0.14278072,"Count":0,"Frequentcy":0},{"Mark":0.152408,"Count":1,"Frequentcy":0.1}],"TaskId":2}